content/blog/controlling-aws-costs-with-lambda-and-pulumi/index.md

```diff
@@ -0,0 +1,366 @@
+---
+date: "2020-04-09"
+title: "Controlling AWS Costs with Pulumi and AWS Lambda"
+authors: ["paul-stack"]
+tags: ["AWS", "serverless", "lambda"]
+meta_desc: "Learn how to use Pulumi and AWS Lambda to create and deploy an application that can control cloud costs"
+meta_image: ""
+---
+
+Due to the nature of the product we build, the Pulumi team needs to have access to several cloud providers to be able to
+develop and test the product. With an increasing number of cloud providers, can come an ever increasing cost
```

Comment:
unnecessary comma?

Reply:
As an Irish man, I should never be discouraged from using grammar ;)